src/SFA.DAS.EmployerFinance.Web/Orchestrators/EmployerAccountTransactionsOrchestrator.cs

```diff
@@ -199,7 +199,7 @@ public EmployerAccountTransactionsOrchestrator(IMediator mediator, ICurrentDateT
                     ExternalUserId = externalUserId
                 });
 
-                var apprenticePaymentGroups = data.Transactions.GroupBy(x => new { x.ApprenticeName, x.ApprenticeNINumber });
+                var apprenticePaymentGroups = data.Transactions.GroupBy(x => new { x.ApprenticeULN,x.ApprenticeName });
```

Comment:
If ApprenticeUln is unique, then is there any value to the secondary grouping of apprentice name

Reply:
No, but I knew of no other way of keeping the name to be passed into line 210